src/__tests__/handleAction-test.js

```diff
@@ -195,4 +196,27 @@ describe('handleAction()', () => {
         .to.deep.equal({ number: 3 });
     });
   });
+
+  describe('with invalid actions', () => {
+    it('should throw a descriptive error when the action object is missing', () => {
+      const reducer = handleAction(createAction('ACTION_1'), identity);
+      expect(() => reducer(undefined))
+      // eslint-disable-next-line max-len
+        .to.throw(/The FSA spec mandates an action object with a type. Try using the createAction\(s\) method./);
```

Comment:
Done. Thanks for the feedback!

`invariant` only throws a regular `Error` and I didn't want to introduce matchers just to check for the error name to be "Invariant Violation". I didn't see it being much value, but the format should be consistent now.